Source/WebCore/css/CSSComputedStyleDeclaration.cpp

```diff
@@ -3196,7 +3198,7 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
         case CSSPropertyGap:
             return getCSSPropertyValuesForShorthandProperties(gapShorthand());
         case CSSPropertyHeight:
-            if (renderer && !renderer->isRenderOrLegacyRenderSVGModelObject()) {
+            if (m_alwaysUseComputedStyle == AlwaysUseComputedStyle::No && renderer && !renderer->isRenderOrLegacyRenderSVGModelObject()) {
```

Comment:
It is not very nice to just sprinkle this to a couple of places when this is basically needed everywhere 'renderer' is accessed. But I suppose you want to do this incrementally?

It would be better to just null the renderer really.